tests/system/python/conftest.py

```diff
@@ -341,11 +348,21 @@ def north_branch(request):
     return request.config.getoption("--north-branch")
 
 
+@pytest.fixture
+def service_branch(request):
```

Comment:
we can skip  this `external_service_branch` for now, No harm to keep.

for notifications world, we can rely on `notify_branch` for corresponding service and plugins